doc/source/structures/misc/steeringmanager.rst

```diff
@@ -0,0 +1,233 @@
+.. _steeringmanager:
+
+SteeringManager
+===============
+
+The SteeringManager is a bound variable, not a suffix to a specific vessel.  This prevents access to the SteeringManager of other vessels.  You can access the steering manager as shown below: ::
+
+    // Display the ship facing, target facing, and world coordinates vectors.
+    SET STEERINGMANAGER:SHOWFACINGVECTORS TO TRUE.
+
+    // Change the torque calculation to multiply the available torque by 1.5.
+    SET STEERINGMANAGER:ROLLTORQUEFACTOR TO 1.5.
+
+.. note::
+
+    .. versionadded:: 0.18
+
+        The :struct:`SteeringManager` was added to improve the accuracy of kOS's cooked steering.  While this code is a significant improvement over the old system, it is not perfect.  Specifically it does not properly calculate the effects of control surfaces, nor does it account for atmospheric drag.  It also does not adjust for asymmetric RCS or Engine thrust.  It does allow for some modifications to the built in logic through the torque adjustments and factors.  However, if there is a condition for which the new steering manager is unable to provide accurate control, you should continue to fall back to raw controls.
+
+
+.. structure:: SteeringManager
+
+    ===================================== ========================= =============
+     Suffix                                Type                      Description
+    ===================================== ========================= =============
+     :attr:`PITCHPID`                      :struct:`PIDLoop`         The PIDLoop for the pitch direction.
+     :attr:`YAWPID`                        :struct:`PIDLoop`         The PIDLoop for the yaw direction.
+     :attr:`ROLLPID`                       :struct:`PIDLoop`         The PIDLoop for the roll direction.
+     :attr:`ENABLED`                       bool                      Returns true if the `SteeringManager` is currently controlling the vessel
+     :attr:`TARGET`                        :struct:`Direction`       The direction that the vessel is currently steering towards
+     :meth:`RESETPIDS()`                   none                      Called to call `RESET` on all steering PID loops.
+     :attr:`SHOWFACINGVECTORS`             bool                      Enable/disable display of ship facing, target, and world coordinates vectors.
+     :attr:`SHOWANGULARVECTORS`            bool                      Enable/disable display of angular rotation vectors
+     :attr:`SHOWTHRUSTVECTORS`             bool                      Enable/disable display of engine thrust vectors
+     :attr:`SHOWRCSVECTORS`                bool                      Enable/disable display of rcs thrust vectors
+     :attr:`SHOWSTEERINGSTATS`             bool                      Enable/disable printing of the steering information on the terminal
+     :attr:`WRITECSVFILES`                  bool                      Enable/disable logging steering to csv files.
```

Comment:
it isnt actually that crazy. All of the basic JSON types we already have

string, number, array, associative array, boolean (kinda). 

we could build a simple enough utility that would allow users to serialize and deserialize simple json files.